src/emqx.app.src

```diff
@@ -1,7 +1,7 @@
 {application, emqx,
  [{id, "emqx"},
   {description, "EMQ X"},
-  {vsn, "4.3.2"}, % strict semver, bump manually!
```

Comment:
sorry, not here.
it should be `include/emqx_release.hrl`

Reply:
I also sent a PR to make it more clear: https://github.com/emqx/emqx/pull/4899

Reply:
@zmstone Could you kick off the CI?